Tests/WebKitTest/WKWebViewExtensionsTests.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  WKWebViewExtensionsTests.swift
+//  SwifterSwift
+//
+//  Created by Tigran Hambardzumyan on 5/23/20.
+//  Copyright © 2020 SwifterSwift
+//
+
+import XCTest
+@testable import SwifterSwift
+
+#if canImport(WebKit)
+import WebKit
+
+final class WKWebViewExtensionsTests: XCTestCase {
+    var webView: WKWebView!
+    let successExpectation = XCTestExpectation(description: "Correct URL")
+
+    override func setUp() {
+        webView = WKWebView()
+    }
+
+    func testLoadURL() {
+        let url = URL(string: "https://www.w3schools.com/")!
+        webView.loadURL(url)
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + 2.0) {
+            guard self.webView.url != nil else {
+                XCTFail("No URL in webView")
+                return
+            }
```

Comment:
You still need to fulfil the expectation so that the test knows to continue. Just assert that the `url` is not `nil`.
```suggestion
            XCTAssertNotNil(self.webView.url)
```

Reply:
The same comment applies to the other tests.

Reply:
+1